homework/index.js

```diff
@@ -29,22 +29,47 @@
     return elem;
   }
 
+  function createLiElement(ul, element1, element2, options1, options2) {
+    const li = createAndAppend('li', ul, {class: 'list'});
+    return createAndAppend(element1, li, options1),
+      createAndAppend(element2, li, options2);
+  }
+
+  function dateChange(date){
+    return new Date(date).toLocaleString();
+  }
+
   function renderRepoDetails(repo, ul) {
```

Comment:
I would have used an HTML table myself of the rendering the repo details, but your approach works too. So I'm fine with it.